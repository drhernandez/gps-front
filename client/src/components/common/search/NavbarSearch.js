import React from "react";
import {
  Form,
  InputGroup,
  InputGroupAddon,
  InputGroupText,
  FormInput
} from "shards-react";
import "../../../styles/search.css";

export default (props) => (
  <Form onSubmit={props.searchFunction} className="main-navbar__search w-100 d-none d-md-flex d-lg-flex">
    <InputGroup seamless>
      <InputGroupAddon type="prepend">
        <InputGroupText>
          <i className="material-icons">search</i>
        </InputGroupText>
      </InputGroupAddon>
      <FormInput
        id="search-input"
        disabled={props.disabled}
        className="navbar-search search_box"
        placeholder={props.placeholder}
      />
    </InputGroup>
  </Form>
);
