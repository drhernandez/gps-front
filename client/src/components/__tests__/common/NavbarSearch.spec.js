import React from 'react'
import renderer from 'react-test-renderer'

import NavbarSearch from '../../common/search/NavbarSearch'

describe('NavbarSearch component', () => {

  it('snapshot', async () => {
    const componet = renderer.create(
      <NavbarSearch />
    )

    expect(componet.toJSON).toMatchSnapshot()
  })
})