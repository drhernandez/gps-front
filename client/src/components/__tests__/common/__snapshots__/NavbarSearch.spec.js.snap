// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarSearch component snapshot 1`] = `[Function]`;
